.github/workflows/test-application.yaml

```diff
@@ -15,7 +15,7 @@ concurrency:
 jobs:
     js-css:
         name: "Node ${{ matrix.node-version }}"
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-20.04
```

Comment:
this should be done on 2.5 branch

Reply:
done https://github.com/sulu/sulu/pull/7615